durexforth.asm

```diff
@@ -97,13 +97,22 @@ _START = * + 1
 
 ; ----------- macros
 
-!set LINK = 0
-
-!macro BACKLINK {
-    ; it's tempting to add the string as a macro parameter,
-    ; but this does not seem to be supported by ACME.
-    !word	LINK
-    !set	LINK = * - 2
+!set TOP = $6fff

```

Comment:
I think it would be ideal to use the area below BASIC ($9fff) as dictionary top. It is unfortunate that the text editor buffer is in the way.